### PR TITLE
Focus color & Dropdown keyboard navigation

### PR DESCRIPTION
**Pull request by Arth101**

### CODE REVIEW

File: collections-online/app/views/mixins/search-filter-section.pug [hunk 1]
```diff
@@ -2,8 +2,8 @@ include ../mixins/date-interval-range-picker
 
 mixin filterSection(displayStringSingular, displayStringPlural, field, type)
   - var dataId = field + '-filters'
-  p.search-filter-sidebar__heading(class='search-filter-sidebar__heading--' + dataId)
-    span(data-action='show-filters' data-id=field tabindex="0")= displayStringSingular
+  p.search-filter-sidebar__heading(class='search-filter-sidebar__heading--' + dataId data-action='show-filters' data-id=field tabindex="0")
+    span= displayStringSingular
```
Comment (hypesystem) [inline collections-online/app/views/mixins/search-filter-section.pug:6]: Any reason to keep the span?

Comment (Arth101) [reply collections-online/app/views/mixins/search-filter-section.pug:6]: Only to avoid rewriting some CSS to keep the arrow icons :)

**Comment by hypesystem** [comment]
Har lavet et bud på search bars :smile: se de commits jeg har smidt op her. Måske skal vi tage en snak med Signe om hvor vigtigt det er at lave det her top nice? For jeg ser en del konflikter med implementationen as is.

F.eks. er topbar-menuen lavet på en måde hvor det er svært at highlighte menupunkterne rigtigt: overflow:hidden skjuler outlinen

**Comment by Arth101** [comment]
I think we are severely restricted by some bad markup and that makes the focus outlines look bad af. You have not shared the designers focus visions with me, but we can easily make the focus outline a little lighter. It is kind of bad for the visually impaired if it is the same pinkish color though.

**Pull request closed without merging**